### PR TITLE
Thunderbird 91 - What's New page

### PR DESCRIPTION
**Pull request by Alecaddd**
This is an updated PR based on the original #269 opened by Ryan.
I decided to continue the work on a new branch as it was easier for me to update and manage.

Here's a screenshot of the page:
91-fullpage

### What to test in this PR
- Animated GIFs play correctly
- Mozilla VPN link works properly
- The page should be fully responsive, with only the header image disappearing at smaller size
- Typos and copy errors
- Visual inconsistencies

### CODE REVIEW

**Comment by ryanleesipes** [comment]
@Alecaddd - Magnus has concerns about the actual data displayed in a couple of the screenshots (showing your conversations with other team members). Do you have alternative data you could display in those?

**Comment by Alecaddd** [comment]
@ryanleesipes sure, I can replace that, no problem

**Comment by Alecaddd** [comment]
@ryanleesipes @Sancus this is ready for a full review.

File: website/thunderbird/91.0/whatsnew/index.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      <aside class="flex-1 lg:mr-8">
+        <img class="w-full h-auto max-w-2xl self-center shadow-img" src="{{ static('/media/img/thunderbird/whatsnew/91/thunderbird-91-calendar-sidebar.gif') }}" alt="Screenshot of the new options in the Thunderbird calendar sidebar.">
+      </aside>
+      <aside class="flex-col flex-1 items-start">
+        <h3 class="header-section">
+          <span>{{ svg('calendar') }}</span>
+          {{ _('Calendar Sidebar')}}
+        </h3>
+        <p class="font-lg tracking-wider leading-loose mb-10">
+          {{ _('Managing multiple calendars has been made easier with the calendar sidebar improvements in this release. There is a quick enable button for disabled calendars, as well as a show/hide icon for easily toggling what calendars are visible. There is also a lock indicator for read-only calendars. Additionally, although not a sidebar improvement, there are now better color accents to highlight the current day in the calendar.') }}
```
Comment (Sancus) [inline website/thunderbird/91.0/whatsnew/index.html:123]: @ryanleesipes This is so awkward lol, can't this section just be named 'Calendar Improvements' and forego explaining the last bit is not a sidebar improvement?

**Review by Sancus — CHANGES_REQUESTED** [review]
Is it intentional that the first section is split and left/right justified, and the following one is centered? This looks a bit odd, as it results in expanding white space in the middle as the screen gets wider eg:
image

File: website/thunderbird/91.0/whatsnew/index.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+          {{ _('The attachments pane has been moved to the bottom of the compose window for better visibility of filenames as well as being able to see many at once. We’ve also added an overlay that appears when you drag-and-drop a file into the compose window asking how you would like to handle the file in that email (such as putting a picture in-line in your message or simply attaching it to the email).') }}
+        </p>
+      </aside>
+      <aside class="flex-1 lg:ml-8">
+        <img class="w-full h-auto max-w-2xl self-center shadow-img" src="{{ static('/media/img/thunderbird/whatsnew/91/thunderbird-91-attachment-overlay.gif') }}" alt="Screenshot of Thunderbird's new attachment overlay.">
+      </aside>
+    </div>
+  </section>
+
+  <!-- UI desnity -->
```
Comment (Sancus) [inline website/thunderbird/91.0/whatsnew/index.html:70]: density :)

File: website/thunderbird/91.0/whatsnew/index.html (hunk 1 above)
Comment (ryanleesipes) [reply website/thunderbird/91.0/whatsnew/index.html:123]: Yeah, I'm okay with changing the name of this section.

File: website/thunderbird/91.0/whatsnew/index.html (hunk 2 above)
Comment (ryanleesipes) [reply website/thunderbird/91.0/whatsnew/index.html:70]: Haha! No - destiny.

Comment (Alecaddd) [reply website/thunderbird/91.0/whatsnew/index.html:70]: "You are my density"
- McFly

**Review by Sancus — APPROVED** [review]

**Comment by mkmelin** [comment]
The point "Support opening of .ics files by double-click in calendar" is confusing as this is really about system integration of the files. I would rephrase it as

"Registers to handle .ics files, letting you preview and import events to the calendar just by double clicking the file"
